perma_web/static/js/folder-tree.module.js

```diff
@@ -37,16 +37,44 @@ export var ls = {
   setCurrent: function (orgId, folderIds) {
     var selectedFolders = ls.getAll();
     selectedFolders[current_user.id] = {'folderIds': folderIds, 'orgId': orgId};
+
+    if (folderIds && folderIds.length){
+      history.pushState(null, null, "?folder=" + folderIds.join('-'));
+    }
+
     Helpers.jsonLocalStorage.setItem(localStorageKey, selectedFolders);
     Helpers.triggerOnWindow("CreateLinkModule.updateLinker");
   }
 };
 
+export function folderListFromUrl() {
+  var queryDict = Helpers.getQueryStringDict();
+  if (queryDict['folder']){
+    try {
+      var folder_list = queryDict['folder'].split('-');
+      folder_list = folder_list.map(s => parseInt(s));
+      folder_list.forEach(i => {if(isNaN(i)) throw "Invalid folder list"});
+      return folder_list
+    } catch (err) {
+      console.error(err);
+    }
```

Comment:
same comment here. Thinking we can punt on this until I fix up the ErrorHandler to be a bit more useful, then I can add to it a sweep for more js errors.

Reply:
Good thinking. Switching!